### DISCUSSION TITLE
iPv6 Identification

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3220] [Development discussions]
Hello,

How can I identify iPv6? I currently have iPv6 installed on my server.

![image](https://github.com/user-attachments/assets/803a15ce-4003-4e79-ad00-b7c48d272cf7)

When I make the above setting, I get the following error.

![image](https://github.com/user-attachments/assets/a923e543-4bd6-4d0d-8bfc-67ecd16470e6)

**Comment by willpower232** [comment]
![image](https://github.com/user-attachments/assets/cd33ea72-fd38-4b8f-ba24-f044fdf2dff2)

I'm afraid that is not an IPv4 address. Do you have one available on your server?

**Reply by TR-SametOzen** [reply]
It doesn't let me skip the iPv4 part blank. What should I write there?

**Reply by willpower232** [reply]
I'm not sure if this feature works without IPv4 I'm afraid
